controller/controller.go

```diff
@@ -313,10 +328,9 @@ func (c *Controller) Unmount(unmountRequest resources.FlexVolumeUnmountRequest)
 	)
 	c.logger.Println(msg)
 
-	return resources.FlexVolumeResponse{
+	return k8sresources.FlexVolumeResponse{
 		Status:  "Success",
 		Message: "Volume unmounted successfully",
-		Device:  "",
```

Comment:
why u remove the Device:""?   because its the default value or something in API was changed?